github/users.go

```diff
@@ -134,6 +134,55 @@ func (s *UsersService) Edit(ctx context.Context, user *User) (*User, *Response,
 	return uResp, resp, nil
 }
 
+// HovercardOptions specifies optional parameters to the UsersService.GetHovercard
+// method.
+type HovercardOptions struct {
+	// SubjectType specifies the additional information to be received about the hovercard.
+	// Possible values are: organization, repository, issue, pull_request. (Required when using subject_id.)
+	SubjectType string `url:"subject_type"`
+
+	// SubjectID specifies the ID for the SubjectType. (Required when using subject_type.)
+	SubjectID string `url:"subject_id"`
+}
+
+// UserContext represents the contextual information about user.
+type UserContext struct {
+	Message *string `json:"message,omitempty"`
+	Octicon *string `json:"octicon,omitempty"`
+}
+
+// Hovercard holds the parsed response from GetHovercard.
+type Hovercard struct {
+	Contexts []*UserContext `json:"contexts,omitempty"`
+}
```

Comment:
It would be good to place `Hovercard` on top of `UserContext`. The general pattern is higher level identifiers should come first. They are easier to understand, and they set context for the lower level identifiers that follow. I.e.:

```Go
// Hovercard ...
type Hovercard struct {
    ...
}

// UserContext ...
type UserContext struct {
    ...
}
```